src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -1835,6 +1838,7 @@ impl<T: AstInfo> AstDisplay for WithOptionValue<T> {
             WithOptionValue::Value(value) => f.write_node(value),
             WithOptionValue::ObjectName(name) => f.write_node(name),
             WithOptionValue::DataType(typ) => f.write_node(typ),
+            WithOptionValue::Secret(name) => f.write_node(name),
```

Comment:
do we need to differentiate this from `WithOptionValue::ObjectName` above so we can correctly construct the statement on restart? Won't all secrets parse as `WithOptionValue::ObjectName` the second time?  Or am I mis-remembering some difference between `AstDisplay` and the actual SQL we use to reload statements when we start up?

Reply:
Oy! Good catch! Just a good old fashioned bug. Fixed now.